be1-go/db/sqlite/cli/schema.sql

```diff
@@ -102,7 +102,7 @@ CREATE TABLE election_valid_vote (
 );
 
```

Comment:
It's great to see all the database code being brought together in one place and this PR significantly improves things, however I also feel the database design could be improved a lot: the naming is not very clear / consistent, to the point where I can't tell if it's not normalized or not.

If my understanding is correct, the following:
* `lao_attendee.attendee_key`
* `election_attendee.attendee_key`
* `election_valid_vote.voter_id`
* `vote_index.voter_id`
can all contain the same value, but their name doesn't show that and the foreign key relationships are not there.

Is it correct to say the following ? 
``` 
vote_index(.voter_id) <-N : 1 -> election_valid_vote(.voter_id) <- 0..1 : 1 -> election_attendee(.attendee_key) <-0..1 : 1 -> lao_attendee.attendee_key
```
Or can valid votes contain voters that are not election_attendees ? 
If so, what's the point of the election_attendees? Is it just the list of subscribers to the election channel? If so, do we really need to persist it ? 

I would consider addressing the fact that the election tables are completely separate from the LAO tables, without any form of foreign key, as that seems wrong.

I would also make sure that, independently of the protocol, and in so far as it makes sense from a data modelling perspective, columns use the same name if they're referring to the same underlying value. It makes the schema much easier to understand and maintain.